pubspec.yaml

```diff
@@ -3,23 +3,23 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 2.0.0-dev.3
+version: 2.0.0-dev.4
 homepage: https://github.com/dart-lang/ffigen
 description: Experimental generator for FFI bindings, using LibClang to parse C header files.
 
 environment:
-  sdk: '>=2.12.0-198.0.dev <3.0.0'
+  sdk: ">=2.12.0-198.0.dev <3.0.0"
```

Comment:
Maybe undo these formatting changes.

If you want to do formatting changes, please make a separate PR and make the GitHub actions check the formatting.